app/controllers/api/v1/api_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::V1::ApiController < ActionController::API
+  before_action :set_default_format
+  before_action :authenticate_token!
+
+   private
+
+     def set_default_format
```

Comment:
sub should be user_id or id, from the looks of it we can name the key anything we want and it should be explanatory i guess.